config/samples/backends/ceph/backend.yaml

```diff
@@ -0,0 +1,60 @@
+# Sample using Ceph for glance, cinder volume, and cinder backup.
+# Requires a running Ceph cluster and its `/etc/ceph` files in secret `ceph-conf-files`
+# This can be achieved with the `ceph` target of `install_yamls`
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  extraMounts:
+    - name: v1
+      region: r1
+      extraVol:
+        - propagation:
+          - CinderVolume
+          - CinderBackup
+          - GlanceAPI
+          extraVolType: Ceph
+          volumes:
+          - name: ceph
+            projected:
+              sources:
+              - secret:
+                  name: ceph-conf-files
+          mounts:
+          - name: ceph
+            mountPath: "/etc/ceph"
+            readOnly: true
+  cinder:
+    template:
+      cinderVolumes:
+        ceph:
+          replicas: 1
```

Comment:
Do we want this to be explicit? It could be eliminated by relying on the default value being 1.

Reply:
I like it explicit, but just because it makes it easier to play with it.
But since that's not the objective of the example we should remove it to leave only what's necessary.